packages/modal/src/presenters/ModalPresenter.js

```diff
@@ -0,0 +1,109 @@
+import React, { Component } from "react";
+import PropTypes from "prop-types";
+import cx from "classnames";
+
+import ModalHeaderPresenter from "./ModalHeaderPresenter";
+import "./ModalPresenter.scss";
+
+export default class ModalPresenter extends Component {
+  static propTypes = {
+    /**
+     * Supports adding any dom content to the body of the modal
+     */
+    children: PropTypes.node,
+    /**
+     * ARIA label attribute for the close button if/when headerChildren
+     * are not utilized
+     */
+    closeButtonAriaLabel: PropTypes.string,
+    /**
+     * Supports adding any dom content to the header of the modal
+     */
+    headerChildren: PropTypes.node,
+    /**
+     * Triggers when one clicks the close button
+     */
+    onCloseClick: PropTypes.func,
+    /**
+     * Triggers when one clicks the overlay behind the modal
+     */
+    onOverlayClick: PropTypes.func,
+    /**
+     * Triggers when one clicks the modal window
+     */
+    onWindowClick: PropTypes.func,
+    /**
+     * Modal is visible when true
+     */
+    open: PropTypes.bool,
+    /**
+     * Title of the modal
+     */
+    title: PropTypes.string,
+    /**
+     * Style of the modal shell
+     */
+    type: PropTypes.string
+  };
+
+  setScrolling = element => {
+    this.hasScrolling = element.scrollHeight > element.clientHeight;
+  };
+
+  render() {
+    const {
+      children,
+      closeButtonAriaLabel,
+      headerChildren,
+      onCloseClick,
+      onOverlayClick,
+      onWindowClick,
+      open,
+      title,
+      type
+    } = this.props;
+
+    const typeKey = type || "standard";
```

Comment:
> `|| "standard"` should be implemented using `defaultProps` instead.
Also, `"standard"` should be referencing a constant e.g. `types.STANDARD`

https://github.com/Autodesk/hig/pull/1092#discussion_r208755296

Reply:
Updated in 85270c27